Seprate Maiden blood spawns and blood orb spawns mistakes

NickSmeding opened the pull request:
The spawned bloods have prio this means if they ever overlap the blood spawn will show and orb bloods not. this is already quite rare but atleast it should not mark 2 mistakes in one tick. This exactly makes quite a bit of sense to split up such that you know what exactly made more bloods orbs spawn. First of all it was coded quite well seprated so it was insanely easy to implement
#9

NickSmeding committed:
Seprate Maiden blood spawns and blood orb spawns mistakes

The spawned bloods have prio this means if they ever overlap the blood spawn will show and orb bloods not. this is already quite rare but atleast it should not mark 2 mistakes in one tick

NickSmeding commented:
<img width="233" height="85" alt="image" src="https://github.com/user-attachments/assets/43dc2f17-7ae0-4d50-963e-5c90db660fac" />

QuestingPet commented on src/main/java/com/tobmistaketracker/TobMistake.java:
```diff
@@ -18,6 +18,7 @@ public enum TobMistake {
     DEATH_XARPUS("Xarpus Death", "death_xarpus.png", "I'm planking!"),
     DEATH_VERZIK("Verzik Death", "death_verzik.png", "I'm planking!"),
     MAIDEN_BLOOD("Maiden Blood", "maiden_blood.png", "I'm drowning in Maiden's blood!"),
+    MAIDEN_BLOOD_BLOB("Maiden Blood (Blob)", "maiden_blood_orb.png", "I'm gonna catch this blob!"),
```
Hmm, I've never heard of it called a "blob" before. That usually means the inferno blobs. How do you feel about changing this to Blood Spawn? I've heard tomato if you prefer that, lol.

NickSmeding replied:
Agreed

NickSmeding committed:
Rename blob to tomato

NickSmeding committed:
Update README.md

if we want to release both changed in same release it should be both under 2.5 when doing the merge/ conflict solves

NickSmeding committed:
Also rename the enum name
